tests/actions/PolicyTagTest.ts

```diff
@@ -0,0 +1,768 @@
+import Onyx from 'react-native-onyx';
+import OnyxUpdateManager from '@libs/actions/OnyxUpdateManager';
+import * as Policy from '@libs/actions/Policy';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {PolicyTags} from '@src/types/onyx';
+import createRandomPolicy from '../utils/collections/policies';
+import createRandomPolicyTags from '../utils/collections/policyTags';
+import * as TestHelper from '../utils/TestHelper';
+import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
+
+OnyxUpdateManager();
+describe('actions/Policy', () => {
+    beforeAll(() => {
+        Onyx.init({
+            keys: ONYXKEYS,
+        });
+    });
+
+    beforeEach(() => {
+        // @ts-expect-error TODO: Remove this once TestHelper (https://github.com/Expensify/App/issues/25318) is migrated to TypeScript.
```

Comment:
This issue is closed. Can we remove this line now? It might result in a TS error, but I think we should solve that.

Reply:
The type of `fetch` isn't the same as `MockFetch` so it still has error if we remove this line.

<img width="978" alt="Screenshot 2024-05-03 at 10 58 07" src="https://github.com/Expensify/App/assets/161821005/8ce35295-211f-4a3c-a844-8ec9689dba15">